contracts/TokenRequest.sol

```diff
@@ -121,10 +182,15 @@ contract TokenRequest is AragonApp {
         }
 
         addressesTokenRequestIds[msg.sender].deleteItem(_tokenRequestId);
+        uint64 refundedDate = getTimestamp64();
 
-        emit TokenRequestRefunded(refundToAddress, refundToken, refundAmount);
+        emit TokenRequestRefunded(_tokenRequestId, refundToAddress, refundToken, refundAmount, refundedDate);
     }
 
+    // function submitTokenRequest(uint256 _tokenRequestId) external auth(SUBMIT_TOKEN_REQUEST_ROLE) {
+    //     finaliseTokenRequest(_tokenRequestId);
+    // }
```

Comment:
Why might we need 2 functions that call `finaliseTokenRequest()` with different permissions? If this unused then remove it.